rasa/nlu/components.py

```diff
@@ -41,10 +41,12 @@ def validate_requirements(component_names: List[Text]) -> None:
         # if available, use the development file to figure out the correct
         # version numbers for each requirement
         raise Exception(
-            "Not all required packages are installed. "
+            "Not all required importable packages are installed. "
             + "To use this pipeline, you need to install the "
             "missing dependencies. "
-            + "Please install {}".format(", ".join(failed_imports))
+            + "Please install the package(s) that contain the module(s){}".format(
```

Comment:
I think a space is missing after `module(s)`

Reply:
Good point! Maybe should also include a colon too. Want to make another quick PR since this one was merged already? :)

Reply:
I've created PR #4162 to fix this.